compact_str/src/features/arbitrary.rs

```diff
@@ -8,6 +8,7 @@ use arbitrary::{
 
 use crate::CompactString;
 
+#[cfg_attr(docsrs, doc(cfg(feature = "arbitrary")))]
```

Comment:
`#[doc()]` is still unstable: https://doc.rust-lang.org/unstable-book/language-features/doc-cfg.html, so it needs to be gated, but since docs.rs uses a nightly rustc by default, this is no problem. When used, you get a message like ["Available on crate feature local only."](https://docs.rs/tzdb/latest/tzdb/fn.local_tz.html). Using a feature instead of a config did not work because I did not manage to make it work in the CI: https://github.com/ParkMyCar/compact_str/blob/57dd13fe44eadf5eca775687837bf3f6db00ce1f/.github/workflows/msrv.yml#L80

Reply:
Sweet! Thanks for improving our docs too then :)